Monika After Story/game/script-windowreacts.rpy

```diff
@@ -586,3 +586,1108 @@ label mas_wrs_twitch:
+           "Hey, [player], if you're trying to find my profile, you're on the wrong site.\n Ahaha! I'm just kidding!",
+           "What's your status update for today? Mine is 'Happily in love!'"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_facebook')
+    return     
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_instagram",
+            category=["Instagram"],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+            },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_instagram:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Gosh, there's quite a few people here that claim to be me...\nI guess I should feel honored, ahaha!",
+           "Any pictures here that catch your eye? Do they have me in them? Ehehe~"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_instagram')
+    return     
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_aseprite",
+            category=["- Aseprite"],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+            },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_aseprite:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Oooh, making some art? Nice! I bet it'll turn out great!",
+           "Making art under design limitations sounds like a challenging task.\n You've got quite some talent, I must say!"    
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_aseprite')
+    return     
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_steam",
+            category=["Steam"],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+            },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_steam:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Looking for some new games to play, [player]?",
+           "Choo choo...Chu~ Ahaha, got you!"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_steam')
+    return     
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_spotify",
+            category=["Spotify"|"iTunes"],
```

Comment:
This will crash you, you're doing logical `OR` for 2 strings, which doesn't make sense. I guess you meant this?
```suggestion
            category=["^(Spotify|iTunes)$"],
```